tests/bluetooth/tester/src/btp/btp.h

```diff
@@ -41,7 +43,9 @@
 #define BTP_SERVICE_ID_AICS	10
 #define BTP_SERVICE_ID_VOCS	11
 #define BTP_SERVICE_ID_PACS	12
-#define BTP_SERVICE_ID_MAX	BTP_SERVICE_ID_PACS
+#define BTP_SERVICE_ID_ASCS	13
+#define BTP_SERVICE_ID_BAP	14
+#define BTP_SERVICE_ID_MAX	BTP_SERVICE_ID_BAP
```

Comment:
I assume that these IDs are specified somewhere, right? 

It seems kind of odd to me that we then define `BTP_SERVICE_ID_MAX` to be the highest value we support, as in theory we could have

```c
#define BTP_SERVICE_ID_BAP 14
#define BTP_SERVICE_ID_TMAP 28
#define BTP_SERVICE_ID_MAX BTP_SERVICE_ID_TMAP
```
and thus the `BTP_SERVICE_ID_MAX` does not really serve any meaningful purpose, does it?

Reply:
currently we have "service_handler[BTP_SERVICE_ID_MAX + 1];" in btp code and since in general new services are added without gaps it is not a big issue ATM.

We can change that in future if bigger gaps show up, although service_handler is just a pointer + count so changing those into eg list wouldn't benefit that much anyway

Reply:
That makes sense. 

what I am thinking is that `BTP_SERVICE_ID_MAX` perhaps should be renamed to `BTP_SERVICE_ID_COUNT` if the intention of it is to provide the number of BTP services implemented. 

The places where we actually use it as a maximum, seems to build on the assumption that MAX = COUNT (which today is true). 

My point is that while this is true today, we are then limited to being able to implement BTP tests in the right order, which is limiting by itself, but it also makes it impossible to implement multiple BTP services in parallel.

Reply:
we don't need to implement those in specific order,  in worst case we will have unused entry in service_handler[] (which is the case already eg with GATT client/server services, as we use only legacy GATT service,  or if building without audio or mesh overlays). And this is already handled by code,  so _MAX indicates maximum value of service ID,  not count

Reply:
Fair point :) 

My worry about this is that we seem to define the BTP as we just go on, and since it's used by multiple projects, it would, IMO, be better to actually define the protocol and then have the implementations implement that protocol, rather than what we are doing now, which is basically defining the protocol as they become used.

Reply:
yeah, unfortunately this is how it goes, the first IUT that implements specified tests works on initial version of BTP at the same time (as you don't really know what is required by PTS and how test is implemented until you run it...)